docs/API/perseo-api.yml

```diff
@@ -0,0 +1,205 @@
+# TODO This is an initial Perseo API description. Incomplete
```

Comment:
I understand these TODO are leaved for next PRs, right?

(No problem with that, just to be sure of the scope of this PR)

Reply:
(Btw, license header missing)

Reply:
Yes. It is difficult for us to describe everything that the API currently returns, this is a job that we should look at together.

License header added in 900fbdc6aa4d27f59c02f4b945ec607df32f512f